rider-fsharp/src/test/kotlin/FSharpCompletionTest.kt

```diff
@@ -8,7 +8,7 @@ import com.jetbrains.rider.test.scriptingApi.*
 import org.testng.annotations.Test
 
 @Test
-@TestEnvironment(toolset = ToolsetVersion.TOOLSET_16_CORE, coreVersion = CoreVersion.DEFAULT)
+@TestEnvironment(toolset = ToolsetVersion.TOOLSET_16_CORE, coreVersion = CoreVersion.DOT_NET_CORE_3_1)
```

Comment:
Is there a reason to use such an old version? Wouldn't it be better to use at least 6? Or `LATEST_STABLE`?

Reply:
Unfortunately, I could not run the tests locally to confirm that it works with dotnet 6.0 due to some weird error `~/Library/Java/JavaVirtualMachines/corretto-11.0.14.1/Contents/Home/bin/java'' finished with non-zero exit value 1`.
I also can see that there are test projects targeting netcoreapp3.1 (and netcoreapp2.1), but let me try if it these do work with dotnet 6 or can even use latest stable version without need to update expectations. Thanks!